sandy/core/cov.py

```diff
@@ -258,12 +259,6 @@ def data(self):
     @data.setter
     def data(self, data):
         self._data = data.astype(float)
-        if not len(data.shape) == 2 and data.shape[0] == data.shape[1]:
-            raise sandy.Error("covariance matrix must have two dimensions")
-        if not np.allclose(data.values, data.values.T):
-            raise sandy.Error("covariance matrix must be symmetric")
-        if (np.diag(data) < 0).any():

```

Comment:
You can delete the checks, for the moment, but write in the docstring that they must be implemented (and which ones)